FAIR-course-UT/.htaccess

```diff
@@ -70,6 +70,22 @@ Redirect 301 /FAIR-course-UT/2025/group1/metadata/metadata_source_distribution h
 
 # GROUP 2
 
+# raw data
+Redirect 301 /FAIR-course-UT/2025-2026/group2/raw_data  https://github.com/s2515091/FAIR_data/tree/main/original_datasets
+
+#fairified data (turtle)
+Redirect 301 /FAIR-course-UT/2025-2026/group2/data#  https://raw.githubusercontent.com/s2515091/FAIR_data/refs/heads/main/turtle_data_of_whole_dataset.ttl
+
+#ontology
+Redirect 301 /FAIR-course-UT/2025-2026/group2/ont https://github.com/s2515091/FAIR_data/blob/main/protege_given_ontology.ttl
+
+#shapes
+Redirect 301 /FAIR-course-UT/2025-2026/group2/shacl https://raw.githubusercontent.com/s2515091/FAIR_data/refs/heads/main/shacl.tll
+
+#metadata (to be published)
+Redirect 301    https://raw.githubusercontent.com/s2515091/FAIR_data/refs/heads/main/metadata.ttl
```

Comment:
Not sure what this will do, but I suspect lack of URL-path param is a problem in this file.